rector.php

```diff
@@ -0,0 +1,55 @@
+<?php
+
+declare(strict_types=1);
+
+use Rector\Caching\ValueObject\Storage\FileCacheStorage;
+use Rector\CodeQuality\Rector\Empty_\SimplifyEmptyCheckOnEmptyArrayRector;
+use Rector\CodeQuality\Rector\Identical\FlipTypeControlToUseExclusiveTypeRector;
+use Rector\CodeQuality\Rector\If_\ExplicitBoolCompareRector;
+use Rector\CodeQuality\Rector\If_\ShortenElseIfRector;
+use Rector\CodingStyle\Rector\FuncCall\ArraySpreadInsteadOfArrayMergeRector;
+use Rector\Config\RectorConfig;
+use Rector\DeadCode\Rector\ClassMethod\RemoveUselessParamTagRector;
+use Rector\DeadCode\Rector\ClassMethod\RemoveUselessReturnTagRector;
+use Rector\DeadCode\Rector\Property\RemoveUselessVarTagRector;
+use Rector\DeadCode\Rector\Ternary\TernaryToBooleanOrFalseToBooleanAndRector;
+use Rector\Php71\Rector\List_\ListToArrayDestructRector;
+use Rector\PHPUnit\PHPUnit100\Rector\Class_\StaticDataProviderClassMethodRector;
+use Rector\Strict\Rector\Empty_\DisallowedEmptyRuleFixerRector;
+
+define('LARAVEL_VERSION', '10.0');
+
+return RectorConfig::configure()
+    ->withPaths([
+        __DIR__ . '/tests',
+    ])
+    ->withAutoloadPaths([
+        __DIR__ . '/vendor/autoload.php',
+    ])
+    ->withBootstrapFiles([
+    ])
+    ->withCache(cacheDirectory: './.rector', cacheClass: FileCacheStorage::class)
+    // register single rule
+    ->withPhpSets(php83: true, php82: true)
```

Comment:
```suggestion
    ->withPhpSets()
```

because,,
```
Run vendor/bin/rector process --config rector.php .php-cs-fixer.dist.php rector.php src/Console/Commands/GenerateRoute.php src/Http/Requests/RequestRuleGeneratorTrait.php tests/FullAccessWrapper.php tests/Unit/FormRequestPropertyHandlerTraitTest.php tests/Unit/RequestRuleGeneratorTraitTest.php --dry-run

Error: ] Pick only one version target in "withPhpSets()". All rules up to this  
         version will be used.                                                  
         To use your composer.json PHP version, keep arguments empty.           

Error: Process completed with exit code 1.
```

Reply:
Thank you for your comment.
I have corrected it in commit [Empty argument to use PHP version of composer.json](https://github.com/litalico-engineering/eg-r2/pull/23/commits/0f48f126673ad05900ea78d386ea60d5cb25a422)